### ISSUE TITLE
[UX] Multiple language contributions

### ISSUE THREAD

**Issue by arthurlutz** [common-voice/common-voice#3884]
I contribute in three languages : French, English and Portuguese. To do this as a user I have to exit the current contribution "Listen/Talk", then switch the language in the interface and then click back on "Contribute". This is a bit cumbersome, it would be nice to have a simpler way : have multiple links for the languages set in the user profile for example. 

An extra cool UX would be to be able to set a mix in the contribute path, listen to 5 french segments, then when accepting another five, switch to the next language, listen to 5 English then 5 Portugese, then 5 French, etc.

Another UX oddity when contributing in multiple languages is that the keyboard shortcuts are all different, so maybe there could be a setting to say "have the interface in main language" (and associated shortcuts don't change). 

Kudos for a great contributor experience by the way !

**Comment by mozgzh** [comment]
Possible solution could be to use the language drop down currently used to select language on the rest of the app which changes the URL to the selected language. (pass the users selected languages so only those are visible)

**Comment by pushkerWebs** [comment]
Hi! I would like to work on this issue. May I please be assigned?

I’ve drafted a possible solution approach and wanted to validate it with maintainers before starting:

Proposed Solution

Preferred Languages in Profile
Add a new profile field (ex : preferredLanguages) allowing users to select multiple languages they contribute in.

Quick Language Switcher on Contribute Page
Display small language buttons (FR / EN / PT) directly on Listen and Talk pages so contributors can switch languages instantly without exiting the session.

Optional Language Rotation Mode
Introduce a setting to automatically rotate languages after a fixed number of segments (ex : 5 per language).

Consistent Keyboard Shortcuts
Add a toggle to keep the UI language fixed to the users main language so keyboard shortcuts dont change when switching contribution languages.

If this approach aligns with the project direction, I’d be happy to implement it.
Thanks!

**Comment by moz-bozden** [comment]
Thank you for the ideas...
Closing this now as stale (hard to implement with the current design, but noted for a future one)
